### ISSUE TITLE
Name node while in "Create New Node" dialog

### ISSUE THREAD

**Issue by robertlugg** [godotengine/godot-proposals#704]
**Describe the project you are working on:**
Running tutorials (dodge the Creeps)

**Describe the problem or limitation you are having in your project:**
Instantiating and naming new nodes requires two steps.

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
In the "Create New Node" dialog, add an additional textbox that shows the name that will be given as well as allow you to change it.

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**
I thought about this because of the way when you create events, you can name the function that receives it in that same dialog

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**
Ease of use minor enhancement

**Is there a reason why this should be core and not an add-on in the asset library?:**
Fundamental operation (creation of node in scene.

**Comment by KoBeWi** [comment]
Well, just after node is created you can press Enter to edit its name. That's a really small additional step.

**Comment by willnationsdev** [comment]
@KoBeWi That's true, but it isn't always plain since users might think they need to click the created node to begin editing the name. And it should be easy enough to implement this feature that it seems worth it to me.

**Comment by bernhardbarker** [comment]
Duplicate of [#1686: Set a node name in the 'Create New Node' dialog box](https://github.com/godotengine/godot-proposals/issues/1686).

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/1686.
